tests/roles/ovn_adoption/tasks/main.yaml

```diff
@@ -121,7 +121,7 @@
         enabled: true
         template:
           ovnNorthd:
-            containerImage: quay.io/podified-antelope-centos9/openstack-ovn-northd:current-podified
+            containerImage: {{ container_registry|default("quay.io") }}/{{ container_namespace|default("podified-antelope-centos9") }}/openstack-ovn-northd:{{ container_tag | default("current-podified") }}
```

Comment:
testing results:

in the periodic/rdo job (https://review.rdoproject.org/r/c/testproject/+/49902/3#message-f227a75e3671428f38029b21aca26ce98f5f2843 )  https://logserver.rdoproject.org/02/49902/3/check/periodic-data-plane-adoption-github-rdo-centos-9-crc-single-node-antelope/c20c09c/controller/pod/ovn-northd-74cb975bc4-2nzgm-describe.txt

```
        *     Image:         quay.rdoproject.org/podified-antelope-centos9/openstack-ovn-northd:e8f50e92676531d00f857753024c2f57
```

and here on the github check/gate it defaults to quay.io and current-podified https://logserver.rdoproject.org/48/148/e580887ae1284a7879f797bbfcff02b41daec325/github-check/data-plane-adoption-github-rdo-centos-9-crc-single-node/5801b70/controller/pod/ovn-northd-b4974776b-5qqbn-describe.txt

```
        *     Image:         quay.io/podified-antelope-centos9/openstack-ovn-northd:current-podified
```